package-lock.json

```diff
@@ -18,6 +18,7 @@
                 "@radix-ui/react-dropdown-menu": "^2.0.6",
                 "@radix-ui/react-icons": "^1.3.0",
                 "@radix-ui/react-popover": "^1.0.7",
+                "@radix-ui/react-scroll-area": "^1.0.5",
```

Comment:
?

Reply:
Resolved in : [2a8989d](https://github.com/covalenthq/goldrush-kit/pull/186/commits/2a8989d5b37ceff28cd8a5c5a782bd96fdc3492c)